pages/osx/mas.md

```diff
@@ -0,0 +1,24 @@
+# mas
+
+> Command line interface for the Mac App Store.
+> Homepage: <https://github.com/mas-cli/mas>.
+
+- Show all installed applications and their product identifiers:
+
+`mas list`
+
+- Search for an application with price included:
```

Comment:
Does this include the price in the search criteria, or does it display the price alongside the results?

Reply:
Display price alongside the results.

Reply:
Right. What about this then?

```suggestion
- Search for an application, displaying the price alongside the results:
```